android/app/build.gradle

```diff
@@ -241,6 +241,7 @@ android {
 
     web {
       dimension 'default'
+      applicationIdSuffix '.web'
```

Comment:
There should be a proper migration notice for existing web users. Otherwise, they may get an unexpected installation/update error and flood our support with questions (or don't update OM at all).

Did you already test how the update/migration procedure will work for existing web users (including those who uses app stores that allow Github apk installation)?